qiita_db/analysis.py

```diff
@@ -619,22 +509,47 @@ def pmid(self, pmid):
         An analysis should only ever have one PMID attached to it.
         """
         with qdb.sql_connection.TRN:
-            self._lock_check()
             sql = """UPDATE qiita.{0} SET pmid = %s
                      WHERE analysis_id = %s""".format(self._table)
             qdb.sql_connection.TRN.add(sql, [pmid, self._id])
             qdb.sql_connection.TRN.execute()
 
-    # @property
-    # def parent(self):
-    #     """Returns the id of the parent analysis this was forked from"""
-    #     return QiitaDBNotImplementedError()
+    # ---- Functions ----
```

Comment:
Nah, it was just old documentation that was there. Removed as I don't think it divides the file between properties and methods anymore...